### ISSUE TITLE
Problems installing because of Guzzle 6

### ISSUE THREAD

**Issue by ruudvdboom** [MultiSafepay/craft-commerce#1]
Can't install the plugin when using other craft plugins like simplonprod/craft-newsletter or when Guzzle 7 is already installed by composer.

Please update php-http/guzzle6-adapter to version 7, so Guzzle 7 wil installed.

See composer log:
```
composer require multisafepay/craft-commerce   
Using version ^1.2 for multisafepay/craft-commerce
./composer.json has been updated
Running composer update multisafepay/craft-commerce
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires multisafepay/craft-commerce ^1.2 -> satisfiable by multisafepay/craft-commerce[1.2.0].
    - multisafepay/craft-commerce 1.2.0 requires guzzlehttp/psr7 ^1.6 -> found guzzlehttp/psr7[1.6.0, ..., 1.8.3] but the package is fixed to 2.1.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require multisafepay/craft-commerce:*" to figure out if any version is installable, or "composer require multisafepay/craft-commerce:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by J0risS** [comment]
Hi @ruudvdboom,
Thanks for reporting this issue! We are looking into a fix for this. We will notify you here once we have released it!

**Comment by ruudvdboom** [comment]
Hi @Joris-MultiSafepay,

Do you have an update on this? 

Ruud

**Comment by J0risS** [comment]
Hi @ruudvdboom,

Yes, we have created the fix internally and we are planning to do a release together with a few other bugfixes somewhere today or tomorrow. 

We will keep you posted, apologies for the delay!

**Comment by J0risS** [comment]
Hi @ruudvdboom,

We have just released version 1.3.0, which should fix your issue. You can find it in the plugin store: https://plugins.craftcms.com/multisafepay, or on github: https://github.com/MultiSafepay/craft-commerce/releases/tag/1.3.0.

Could you please check and confirm this fixes your issue? Thanks!

**Comment by ruudvdboom** [comment]
Thanks, installing now works. 
But I've found another bug. I will create a new issue for that.
